scripts/ghutils.py

```diff
@@ -436,12 +436,17 @@ def create_release(repo, version, bodyfile):
     # Store version in context for use in notifications
     ctx_obj.version = version
     try:
-        release = repo.create_release(
-            tag_name=version,
-            name=version,
-            body=release_notes,
-        )
-        logger.info("Release {} created.".format(version))
+        try:
+            release = [r for r in repo.iter_releases() if r.name == version][0]
```

Comment:
It might be worth rerunning this to take account of situations where something has changed or at least expose any difference.

Reply:
OK, I'll check the body matches as well.

Reply:
Done